js/on-demand-scripts/qg-modal.js

```diff
@@ -16,37 +16,40 @@
         link.removeClass('lightbox');
       } // Detect in-page content modals
   
-  
-      $('a.lightbox[href^="#"]').each(function (e) {
-        $(this).addClass('content-modal');
-      }); // Detect images modals
-  
-      $('a.lightbox[href$=".png"],a.lightbox[href$=".jpg"],a.lightbox[href$=".gif"]').each(function (e) {
-        $(this).addClass('image-modal');
-      }); // Use bootstrap modals for anything with the 'lightbox' class
-  
-      $('a.lightbox').each(function (e) {
-        // Handle ajax modals
-        if (!$(this).hasClass('image-modal') && !$(this).hasClass('content-modal')) {
-          $(this).addClass('ajax-modal');
-          var ajaxUrl = $(this).attr('href');
-  
-          if (ajaxUrl.indexOf('?') !== -1) {
-            ajaxUrl += '&SQ_ASSET_CONTENTS&SQ_PAINT_LAYOUT_NAME=modal';
-          } else {
-            ajaxUrl += '?SQ_ASSET_CONTENTS&SQ_PAINT_LAYOUT_NAME=modal';
+      if(!$('a.help').attr('class')) {
```

Comment:
from what i see `(!$('a.help').attr('class'))` is always equally false if $('a.help') exists
as the selecter 'a.help' means this 'a' tag always has a class attribute (which contains the class name `help`)

you could use `($('a.help').length === 0)` if you want to check is the element exist or not

Reply:
Updated anchor tag check to include .not('.help')